src/timebar/components/playback.module.css

```diff
@@ -44,7 +44,7 @@
   margin: 0 2.4rem;
 }
 
-@media only screen and (max-width: 72rem) {
+@media only screen and (max-width: var(--timebar-compact-breakpoint)) {
```

Comment:
I don't think this works, please review: https://stackoverflow.com/a/40723269/8332192

Reply:
Thanks, will check this